src/main/java/me/tigerhix/lib/scoreboard/ScoreboardLib.java

```diff
@@ -24,25 +27,43 @@ public static void setPluginInstance(Plugin instance) {
     ScoreboardLib.instance = instance;
   }
 
-  public static Scoreboard createScoreboard(Player holder) {
+  public static Scoreboard createScoreboard(Player holder, @Nullable String new_board_name) {
+    String board_name;
+    if (new_board_name!=null && new_board_name.length()>0) {
+      board_name=new_board_name;
+    } else {
+      int leftLimit = 48; // numeral '0'
```

Comment:
This logic is something that would have a better place in some sort of utility file

Reply:
ok I will try to do it :) after Christmas probably.